.fixtures.yml

```diff
@@ -7,11 +7,13 @@ fixtures:
     yumrepo: "puppetlabs/yumrepo_core"
     apt: "puppetlabs/apt"
   repositories:
+    bootstrap: 'https://github.com/puppetlabs/puppetlabs-bootstrap'
+    deploy_pe: 'https://github.com/jarretlavallee/puppet-deploy_pe'
     facts: 'https://github.com/puppetlabs/puppetlabs-facts'
-    puppet_agent: 'https://github.com/puppetlabs/puppetlabs-puppet_agent'
     provision: 'https://github.com/puppetlabs/provision'
-    bootstrap: 'https://github.com/puppetlabs/puppetlabs-bootstrap'
+    puppet_agent: 'https://github.com/puppetlabs/puppetlabs-puppet_agent'
     puppet_conf: 'https://github.com/puppetlabs/puppetlabs-puppet_conf'
     ruby_task_helper: 'https://git@github.com/puppetlabs/puppetlabs-ruby_task_helper'
     stdlib: 'https://github.com/puppetlabs/puppetlabs-stdlib'
-    deploy_pe: 'https://github.com/jarretlavallee/puppet-deploy_pe'
+  symlinks:
```

Comment:
Thanks for the contribution, looks good!  I only had a question about this `symlinks` entry, what does this do?  I was able to run the unit tests with `pdk test unit` with and without this, so I think it should set up everything related to this module.  Let me know if I'm missing something, I haven't used `symlinks` in fixtures before so it's entirely possible.

Reply:
Hello! I was working around needing `influxdb_setup` type available to: `is_expected.to contain_influxdb_setup(Facter.value(:networking)['fqdn'])` in `spec/classes/init_spec.rb`

Starting to wonder if my pdk is/was misconfigured. I couldn't get it to pass without symlink to itself.

Reply:
usually puppetlabs_spec_helper creates the symlink, thats why we removed the symlink option from a lot of modules 🤔

Reply:
Shall I resubmit without the additional symlink, not to be confused with title of PR.  :)